diff -uNr SOAP-Lite-0.55-orig/lib/SOAP/Transport/HTTP.pm SOAP-Lite-0.55/lib/SOAP/Transport/HTTP.pm
--- SOAP-Lite-0.55-orig/lib/SOAP/Transport/HTTP.pm	Fri May  3 16:22:19 2002
+++ SOAP-Lite-0.55/lib/SOAP/Transport/HTTP.pm	Fri May  3 16:25:43 2002
@@ -390,6 +390,8 @@
   while (my $c = $self->accept) {
     while (my $r = $c->get_request) {
       $self->request($r);
+      # Handle basic authentication headers
+      $ENV{'HTTP_authorization'} = $r->header('Authorization');
       $self->SUPER::handle;
       $c->send_response($self->response)
     }
